draft-davidben-tls-merkle-tree-certs.md

```diff
@@ -225,7 +227,13 @@ when, and only when, they appear in all capitals, as shown here.
 
 This document additionally uses the TLS presentation language defined in {{Section 3 of !RFC8446}}.
 
-# Overview
+## Time {#time}
+
+All time computations in this document are represented by POSIX timestamps, defined in this document to be integers containing a number of seconds since the Epoch, defined in Section 4.16 of {{POSIX}}. That is, the number of seconds after 1970-01-01 00:00:00 UTC, excluding leap seconds. A UTC time is converted to a POSIX timestamp as described in {{POSIX}}.
+
+Durations of time are integers, representing a number of seconds not including leap seconds. They can be added to POSIX timestamps to produce other POSIX timestamps.
```

Comment:
Do you allow negative integers?

Reply:
Hmm, not sure. We don't currently use them anywhere, but it's perfectly defined to add integers in integer space. Dunno, do you have a preference?

Reply:
remember that time_t is signed.

Reply:
Although this spec talks in terms of mathematical integers anyway. But yeah it seems signed is easier here.